contracts/Docs.sol

```diff
@@ -158,13 +158,16 @@ abstract contract Docs is ImmutableState, Restrictable {
     }
 
     ///
-    function getDocByTime(bytes32 ehrID, IDocs.Type docType, uint32 timestamp)
-        public view returns (IDocs.DocumentMeta memory)
+    function getDocByTime(bytes32 ehrID, DocType docType, uint32 timestamp)
+    public view returns (DocumentMeta memory)
```

Comment:
please add addition offset

Reply:
Now the type description has been moved to IDocs, so the correct way is
IDocs.Type
IDocs.DocumentMeta